frontend/src/layouts/EditPublishLayout.tsx

```diff
@@ -1,11 +1,11 @@
 import React, { useRef } from 'react';
-import ReactQuill from 'react-quill';
+import ReactQuill,{Quill} from 'react-quill'; 
```

Comment:
linting issue. 
Please run the following commands once.
```bash
npm run lint-fix
npm run format
```

Reply:
Have run this command, and haven't encountered any errors so far.